oauth-jwt/jwt-auth-server/pom.xml

```diff
@@ -155,22 +81,6 @@
 
     <build>
         <plugins>
-            <plugin>
```

Comment:
should we keep this config? I don't think it's related to the Keycloak update, and it's shown in the article as well: https://www.baeldung.com/spring-security-oauth-jwt#maven

Reply:
@lor6 - yeah, we should. I added it back.